lib/ttt.ex

```diff
@@ -4,19 +4,31 @@ defmodule TTT do
     Setup.new_game()
   end
 
-  def play(board, game, status, %Player{name: _name, token: _token, type: type} = turn)
+  def make_a_play(board, game, _status, %Player{name: _name, token: _token, type: type} = turn)
```

Comment:
Do you need the `when type == :whatever` checks here? I thought that you could use pattern matching in the arguments to filter your type?